service/db.go

```diff
@@ -1,92 +1,85 @@
 package service
 
 import (
+	"database/sql"
 	"fmt"
+	pb "github.com/hwsc-org/hwsc-api-blocks/int/hwsc-user-svc/proto"
 	log "github.com/hwsc-org/hwsc-logger/logger"
 	"github.com/hwsc-org/hwsc-user-svc/conf"
-	"github.com/mongodb/mongo-go-driver/mongo"
-	"golang.org/x/net/context"
+	"google.golang.org/grpc/codes"
+	// database/sql uses this library indirectly
+	_ "github.com/lib/pq"
 	"os"
 	"os/signal"
 	"syscall"
 )
 
-var (
-	mongoClientReader *mongo.Client
-	mongoClientWriter *mongo.Client
+const (
+	dbDriverName = "postgres"
 )
 
-const (
-	mongoReader = "mongo reader"
-	mongoWriter = "mongo writer"
+var (
+	connectionString    string
+	postgresDB          *sql.DB
+	postgresUnavailable *pb.UserResponse
 )
 
 func init() {
-	log.Info("Connecting to mongo databases")
+	postgresUnavailable = &pb.UserResponse{
```

Comment:
Wondering if leave this here, or put it under error.go file, or not use it all

Reply:
This block is not needed because the service will stop on startup if it cannot connect to the db.
```
	postgresUnavailable = &pb.UserResponse{
		Status:  &pb.UserResponse_Code{Code: uint32(codes.Unavailable)},
		Message: codes.Unavailable.String(),
	}
```

Reply:
It was more like a global error response, like how we define errors in error.go.
Every service that needs to access db, i will need to refresh my connection, and i did not want to rewrite this same error response over and over again. It was to remove redundancy.

Should I still remove it?